.github/workflows/data-builder-container.yml

```diff
@@ -1,15 +1,16 @@
 name: Build and push the data builder image to GitHub Container Registry
 on:
   push:
-    tags:
-      - 'v*.*.*' # i.e. has to have a release tag to run 
+    branches:
+      - prod # aka prod/stable
```

Comment:
I would explicit those comments, because even as an insider it bends my neurons. 
Suggestions:
-  Branch to be deployed in production, through buillds tagged `prod`
-  Branch to be deployed in development, through builds tagged `dev`

Reply:
Fair point, I've now added this to each workflow file to be more clear in commit 807fa1b.